lib/grpc/server/adapters/cowboy/handler.ex

```diff
@@ -69,8 +69,10 @@ defmodule GRPC.Server.Adapters.Cowboy.Handler do
         local: opts[:local],
         codec: codec,
         http_method: http_method,
+        http_transcode: access_mode == :http_transcoding,
```

Comment:
I would do the test somewhere else and here I would just assign the resulting value

```elixir
       # in some where: http_transcode = access_mode == :http_transcoding
       
        # and here just
        http_transcode: http_transcoding,
```

Reply:
Done in b2f4598